src/SpectatorView.Unity/Assets/SpectatorView/Scripts/StateSynchronization/NetworkedComponents/DynamicGameObjectHierarchy/DynamicGameObjectHierarchyBroadcaster.cs

```diff
@@ -85,30 +85,30 @@ protected override void ProcessNewConnections(IEnumerable<SocketEndpoint> connec
             }
         }
 
-        protected override void SendCompleteChanges(IEnumerable<SocketEndpoint> endpoints)
+        protected override void SendCompleteChanges(IEnumerable<INetworkConnection> connections)
         {
-            foreach (SocketEndpoint endpoint in endpoints)
+            foreach (INetworkConnection connection in connections)
             {
                 PerConnectionInstantiationState state;
-                if (!perConnectionInstantiationState.TryGetValue(endpoint, out state))
+                if (!perConnectionInstantiationState.TryGetValue(connection, out state))

```

Comment:
Not sure what's going on here. Either
`if (!perConnectionInstantiationState.ContainsKey(connection))`
or there is a bug and `state` was supposed to be used somehow.

See similar usages of TryGetValue

Reply:
i think this is just old code, i didn't touch this implementation but will move towards ContainsKey